spec/lib/tasks/after_tasks.rake_spec.rb

```diff
@@ -524,11 +524,13 @@
   end
 end
 
-describe "rake After:remove_roles" do
-  it "remove translation admin role if exists" do
+describe "rake After:remove_translation_admin_role" do
```

Comment:
We can shorten this -- we don't need to check that the role was created, nor do we need to call `save!` on the user when adding the role (although you're right we'd need to save in most cases), but we do need to check that the role is removed from the user:

```
    user = create(:user)
    user.roles = [Role.create(name: "translation_admin")]
    expect(Role.all).to be_empty
    expect(user.reload.roles).to be_empty
```

Reply:
done